client/src/components/UI/Toast.tsx

```diff
@@ -0,0 +1,58 @@
+import React,{useCallback, useEffect} from "react";
+import { toastActions } from "../../store/toast";
+import { useDispatch } from "react-redux";
+import styles from "./Toast.module.css";
+
+export interface IToastProps {
+    id: number;
+    title: string;
+    description: string;
+    backgroundColor: string;
+}
+
+type toastProps = {
+    toastList: IToastProps[];
+    position: string;
+}
+
+const Toast : React.FC<toastProps> = ({ toastList, position }) => {
+
+  const dispatch = useDispatch();
+  const deleteToast = useCallback((id:number) => {
+      const toastListItem = toastList.filter((e) => e.id !== id);
+      dispatch(toastActions.removeToast(toastListItem));
+    },
+    [toastList,dispatch]
+  );
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (toastList.length) {
+        deleteToast(toastList[0].id);
+      }
+    }, 3000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, [toastList, deleteToast]);
+  return (
+    <div className={`${styles.container} ${styles[position]}`}>
+      {toastList.map((toast, i) => (
+        <div
+          key={i}
+          className={`${styles.notification} ${styles.toast} ${styles[position]}`}
+          style={{ backgroundColor: toast.backgroundColor }}
+        >
+          {toast.description !=='success' ? <button onClick={() => deleteToast(toast.id)}>X</button> : null}
```

Comment:
why checking for success,. the closable option can be passed as props and if its true we show the cross